add-ons/fabric8/fabric8.addon

```diff
@@ -34,4 +34,4 @@ echo Or you can watch in the OpenShift console via:
 echo   minishift console
 echo -
 echo Then you should be able the open the fabric8 console here:
-echo   https://fabric8-fabric8.#{routing-suffix}/
+echo   http://fabric8-fabric8.#{routing-suffix}/
```

Comment:
@kadel Does route created based on http instead of https for this addon?

Reply:
[Fabric8 Route](https://github.com/fabric8io/fabric8-resources/blob/8e26375977614fb8a833f1514c2a442fc562c8bf/fabric8-system-openshift.yml#L2732) is HTTP only.

Reply:
Ah right,

Reply:
👍